bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -0,0 +1,234 @@
+      </div>
+  </section>
+
+  <section>
+    <ul class="mobile-download-buttons mobile-pocket">
+      <li class="android">
+        {{ google_play_button(href='https://app.adjust.com/h5iycst', id='playStoreLink') }}
+      </li>
+      <li class="ios">
+        <a id="appStoreLink" href="https://app.adjust.com/h5iycst" data-link-type="download" data-download-os="iOS">
+          <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+        </a>
+      </li>
+    </ul>
+  </section>
+
+  <section>
+    <ul class="mobile-download-buttons mobile-lockbox">
+      <li class="ios">
+        <a id="appStoreLink" href="https://app.adjust.com/kq8zhhz" data-link-type="download" data-download-os="iOS">
+          <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+        </a>
+      </li>
+    </ul>
+  </section>
+
+  <section>
+    <ul class="mobile-download-buttons mobile-notes">
+      <li class="android">
+        {{ google_play_button(href='https://play.google.com/store/apps/details?id=org.mozilla.testpilot.notes') }}
+      </li>
+    </ul>
+  </section>
+</aside>
+
+<div id="bottom-banner">
+  <div class="mzp-l-content">
+    <div class="banner-title">
+      <p>{{ _('You deserve peace of mind everywhere.') }}</p>
+    </div>
+  </div>
+  <div id="bottom-cta">
+    <div id="fxa-sign-up-form">
+      <div class="fxa-cta show-fxa-supported-signed-in show-fxa-supported-signed-out">
+        <div class="show-fxa-supported-signed-in">
+          <ul class="mobile-download-buttons" id="bottom-banner-mobile">
+            <li>
+              {{ google_play_button(id=android_id, extra_data_attributes={'download-location': button_location}) }}
+            </li>
+            <li>
+              <a href="{{ firefox_ios_url(app_store_campaign) }}" data-link-type="download" data-download-os="iOS" data-download-location="{{ button_location }}" id="{{ ios_id }}">
+                <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="show-fxa-supported-signed-out">
+          {{ fxa_email_form(entrypoint='accounts-page', utm_source='accounts-page', utm_params={'campaign': 'fxa-embedded-form', 'content': 'accounts-page', 'medium': 'referral'}, button_class='button button-blue', button_text=_('Create an Account')) }}
+          <div id="fxa-sign-in">
+            <p>{{_('Already have an account?')}} <a href="#">{{ _('Sign In') }}</a></p>
+          </div>
+        </div>
+      </div>
+      <div class="mobile-ctas">
+        <span class="show-fxa-ios">
+          <a class="button button-hollow button-light" rel="external" href="https://support.mozilla.org/kb/sync-firefox-bookmarks-and-browsing-history-iOS">{{ _('Set up Sync for iOS') }}</a>
```

Comment:
:( do you think I should call this out specifically with the `html.ios` selector in .scss?

Reply:
I have `css/base/mozilla-fxa-state.scss` in the static bundle for `firefox_accounts_2018` :(
maybe I should go ahead and hide it myself under the `html.ios` css selector?

Reply:
I am pretty sure I had to write some custom JS to get the mobile stuff to work on the old benefits page because it's detected separately from the fxa stuff but now I can't *find* it. Don't do anything for now, I'll keep looking. Or let me know if you solve it first.

Reply:
Ah! I think this is a display bug. I suspect the button is visible but off screen:

<img width="394" alt="screen shot 2018-11-28 at 8 57 24 pm" src="https://user-images.githubusercontent.com/854701/49200139-41b34100-f350-11e8-9aee-e237a0b14afb.png">

Reply:
Do you think you could swap this custom code out for a [Protocol compact call out](https://protocol.mozilla.org/patterns/organisms/call-out.html#call-out-compact)?

You can't use the macro because OMG custom code but maybe you can still use the component by hard coding the HTML onto the page.

I am thinking that might solve some of the cross browser layout stuff but if you think that's more work than just tweaking the custom component do what you think is best :)